mlir/include/Quantum/IR/QuantumOps.td

```diff
@@ -522,7 +522,8 @@ def MeasureOp : Quantum_Op<"measure"> {
     }];
 
     let arguments = (ins
-        QubitType:$in_qubit
+        QubitType:$in_qubit,
+        OptionalAttr<I8Attr>:$postselect
```

Comment:
Probably here you can remove the optional as you always have -1, 0,1

Reply:
Done